peppy/project.py

```diff
@@ -151,6 +157,11 @@ def __init__(
 
         self.name = self.infer_name()
         self.description = self.get_description()
+
+        if pd_object is not None:
```

Comment:
```suggestion
        if pd_object:
```

Reply:
It won't work! if pd_object will be pandas object, it will through an error

Reply:
![image](https://user-images.githubusercontent.com/41573628/185235607-fb2ebbc4-57a7-427f-9935-3b1733c079e9.png)

Reply:
Ok, weird but that's how pandas works apparently :D

Reply:
More discussion on these things here: https://stackoverflow.com/questions/7816363/if-a-vs-if-a-is-not-none